plugins/obs-outputs/CMakeLists.txt

```diff
@@ -179,6 +179,22 @@ elseif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/ftl-sdk/CMakeLists.txt")
       obs-outputs PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/ftl-sdk/libftl/posix)
   endif()
 
+  if(NOT OS_WINDOWS)
```

Comment:
This sounds like a mouthful that should be fixed in code and not just ignored.

Reply:
I don't think someones want to fork ftl-sdk and fix all this.

Reply:
Ah, I didn't notice the indent, this is for FTL.. Bummer.